libs/elements/src/lib/form-validation/vl-form-validation.element.ts

```diff
@@ -67,6 +69,29 @@ export const vlFormValidationElement = (SuperClass: Class): Class => {
             }
         }
 
+        /**
+         * resets validity van individueel form element
+         */
+        resetValidity(): void {
```

Comment:
In vl-form-validation.lib.js staat deze code ook (resetInput() methode), kan je die methode aanspreken ipv ze te dupliceren?

Reply:
is een goed voorstel; aangepast

origineel is die functie ook niet publiekelijk toegankelijk (soort van protected) daarom dat ik die initieel gekopïeerd had maar ik denk dat het idd beter is die functie maar 1 keer te hebben dus heb die nu toegankelijk gemaakt.

binnen resetValidity gebruik ik nu de resetInput methode, mss niet slecht dat die namen verschillen:
- resetInput aanvaard een HTMLElement
- resetValidity word aangeroepen op het element zelf
kan het ook 2 keer dezelfde naam geven, het is alleen dat de signatuur van methodes dan verschillen afhankelijk waar het gebruikt wordt (`vl.formValidation` of element dat erft van `vlFormValidationElement`)

aan de andere kant, resetInput vind ik ook geen goeie naam (want het reset enkel de validatie, niet de value) maar mss wel goed dat originele naam word behouden?

Reply:
dit was mondeling besproken met Kristof - merging